lib/plugin.js

```diff
@@ -148,7 +148,7 @@ export default fp(async function (fastify, { prefix = "/", extensions, cwd = pro
             const template = html`<${tag} version="${version}" locale='${locale}' translations='${translations}' initial-state='${initialState}'></${tag}>${null} `;
             const hydrateSupport =
               mode === "hydrate"
-                ? f.script(joinURLPathSegments(prefix, 'node_modules/@lit-labs/ssr-client/lit-element-hydrate-support.js'), { dev: true })
+                ? f.script(joinURLPathSegments(prefix, 'node_modules/@lit-labs/ssr-client/lit-element-hydrate-support.js'), { dev: development })
```

Comment:
I know it's been ages and ages since this change, but I'm seeing this script included on https://dev.finn.no/ now that the footer podlet is set to `hydrate`. The check for `!development && dev` in `script.js` is essentially short-circuited with this when `development === false`, so the script is always included.

1. Do we need the podlet-server to include this script?
2. If so, how do we properly format the URL to something like Eik?

Reply:
In dev mode, this script is fetched dynamically via a special dev on the fly bundling endpoint for `node_modules`. When the app makes a request to it, Esbuild bundles the file, caches it (IIRC) and then responds with it.

In prod mode, we prebundle these deps so that when you run the build command you end up with a bundle that includes this.